frontend/src/components/PLAYGROUND/ReportSelector/ReportParameters/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { FormInput } from "components/FormInput";
+import React from "react";
+// todo: Define a good type. "Any" type temporarily permitted.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const ReportParameters = ({ values, setFieldValue, categoriesAndTypes }: any) => {
+  const renderComponent = (parameterName: string) => {
+    switch (parameterName) {
+      case "fiscal":
+        return (
+          <FormInput
```

Comment:
I thought about suggesting a re-usable component for this, but it would probably take up just as much space lol

Reply:
could be made more efficient by adding a config that this uses to build the inputs